src/com/owncloud/android/services/observer/FileObserverService.java

```diff
@@ -99,7 +100,11 @@
     public static void initialize(Context context) {
         Intent i = new Intent(context, FileObserverService.class);
         i.setAction(ACTION_START_OBSERVE);
-        context.startService(i);
+        if (Build.VERSION.SDK_INT > Build.VERSION_CODES.N_MR1) {
```

Comment:
The problem you report appears in In API 26 and higher, but N_MR1 is API 25, it should be O, which is API 26

Reply:
Ups, forget my comment, just seen you are using > instead of >=